docs/user/build-customization.rst

```diff
@@ -218,3 +223,36 @@ To setup it, you need to define the version of Node.js to use and install the de
          - npm ci
          # Install any other extra dependencies to build the docs
          - npm install -g jsdoc
+
+
+Override the build process completely
```

Comment:
This is going to be hidden under all the examples. I'd argue this is a different page.